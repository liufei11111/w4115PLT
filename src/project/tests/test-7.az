Int main(Int argc, String argv){
Int a;
Boolean b1;
b1 = true;
for (a = 0; a < 10; a = a +1 ){
	Int b;
	b =1;
}
for (a = 0; a < 10; a = a +1 )
	Int c;

for (a = 0; b1; a = a +1 ){
	Int d;
	d =1;
}
}

