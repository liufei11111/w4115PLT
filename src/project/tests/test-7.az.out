
public static int main(int argc, String argv)
{
try{int a=0;
boolean b1=false;
b1 = True;
for (a = 0 ; a < 10 ; a = a + 1) {
  int b=0;
  b = 1;
}
for (a = 0 ; a < 10 ; a = a + 1) int b=0;
for (a = 0 ; b1 ; a = a + 1) {
  int b=0;
  b = 1;
}
}catch(Exception e){return -1;}}

